profiles/kentik_snmp/cisco/cisco-all-devices.yml

```diff
@@ -1146,5 +1148,25 @@ metrics:
           name: cHsrpGrpStandbyRouter
           OID: 1.3.6.1.4.1.9.9.106.1.2.1.1.14
           conversion: hextoip
+  - MIB: CISCO-AUTH-FRAMEWORK-MIB 
+    # The table contains a list of authentication session  
+    table:
+      OID: 1.3.6.1.4.1.9.9.656.1.4.1
+      name: cafSessionTable
+    symbols:
+        # Indicates the session timeout used by Authentication Framework in the authentication session    
+      - OID: 1.3.6.1.4.1.9.9.656.1.4.1.1.15
+        name: cafSessionTimeout
+    # A list of port entries. An entry will exist for each interface which support Authentication Framework feature."       
+    table:
+      OID: 1.3.6.1.4.1.9.9.656.1.2.1
+      name: cafPortConfigTable
+    symbols: 
+        # Specifies if reauthentication is enabled for this port.
+      - OID: 1.3.6.1.4.1.9.9.656.1.2.1.1.6
+        name: cafPortReauthEnabled
+        # Specifies the reauthentication interval, after which the port will be reauthenticated if value of the corresponding instanceof cafPortReauthEnabled is 'true'
+      - OID: 1.3.6.1.4.1.9.9.656.1.2.1.1.7
+        name: cafPortReauthInterval
```

Comment:
this isn't formatted correctly for a tabular MIB; there should be a single table with symbols that will be the numeric metric values; and optionally metric_tags that will be decorated attributes on the aforementioned metrics.

Reply:
I've used metric_tags for the OID which isn't in the table.